src/main/java/org/opensearch/ad/ml/ModelIdMapper.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.ml;
+
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utilities to map between single-stream models and ids.  We will have circuilar
```

Comment:
This class only used for single-flow detector models? If that's the case, how about rename the class as like "SingleFlowModelIdMapper" ?

Reply:
renamed